README.md

```diff
@@ -26,11 +26,20 @@ default['supermarket_omnibus']['package_repo'] = 'chef-current'
 default['supermarket_omnibus']['package_url'] = 'http://bit.ly/98K8eH'
 ```
 
+If you wish to use custom SSL certificates, define the following `config` attributes:
+
+```ruby
+default['supermarket_omnibus']['config']['ssl']['certificate'] = '/full/path/to/certificate.crt'
+default['supermarket_omnibus']['config']['ssl']['certificate_key'] = '/full/path/to/certificate.key'
+```
+
+To see all supermarket `config` attributes you can override, see `/etc/supermarket/supermarket-running.json`
```

Comment:
Hey @alexpop I recommend you link to this instead:  https://github.com/chef/omnibus-supermarket/blob/master/cookbooks/omnibus-supermarket/attributes/default.rb